cves/kernel/CVE-2021-4154.yml

```diff
@@ -151,10 +150,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerablity was discoverd and reported by the syzkiller fuzzer, an 
+    unsupervised coverage-guided kernal fuzzer.
```

Comment:
I like the description on what this fuzzer actually does

Reply:
Thank you!